net/Kconfig

```diff
@@ -146,6 +146,13 @@ config NET_LL_GUARDSIZE
 		the L2/L3 (MAC/IP) data on Network layer, which will be beneficial
 		to L3 network layer protocol transparent transmission and forwarding
 
+config NET_JUMBO_FRAME
```

Comment:
do we have the jumbo frame user case on real hardware device?

Reply:
Yes, the devices also require that the  data to be sent should be in the continuous linear space.

Reply:
I think we should redesign the management of network memory pools, such as Linux kernel sk_buffer. Such evolution is not a long-term solution，just my complaint and not related to this PR.